cmake/config/mac-clang.txt

```diff
@@ -197,6 +197,14 @@ macro (link_against_seexpreditor target)
     target_link_libraries (${target} ${SEEXPREDITOR_LIBRARIES})
 endmacro ()
 
+macro (link_against_embree target)
+    if (USE_EXTERNAL_EMBREE)
+        target_link_libraries (${target} ${EMBREE_LIBRARIES})
+    else ()
+        #TODO
```

Comment:
Please do.

Reply:
Keep it here since I don't have mac at my disposal